source/language/pulses.rst

```diff
@@ -38,47 +38,57 @@ interchangeable at the pulse level. Due to varying physical qubit
 properties a microcode definition of a gate on one qubit will not
 perform the equivalent operation on another qubit. To meaningfully
 describe gates as pulses we must bind operations to specific qubits.
-QASM achieves this by prefixing qubit references with ``%`` to indicate
-a specific qubit on the device, e.g. ``%2`` would refer to physical
-qubit 2, while ``%q`` is an unbound reference to a physical qubit.
+QASM achieves this by prefixing qubit references with ``$`` to indicate
+a specific qubit on the device, e.g. ``$2`` would refer to physical
+qubit 2.
+
+One can define a `defcal` using an arbitrary `$` identifier, provided that gate is called using physical
+qubits. For instance, to define an equivalent `rz` calibration on qubits 0 and 1, we could write
+
+.. code-block:: c
+
+   defcal rz(angle[20]:theta) $q { ... }
```

Comment:
The example would be clearer with some whitespace between the `defcal` and the uses below. even inserting a line with a comment would help. i.e.
```
// we've defined 'rz' on arbitrary physical qubits, so we can already do:
...
```

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/150/commits/7ba67251679c80311bc500d812ce712fff8e1b24.